menus/pause_menu.gd

```diff
@@ -1,11 +1,28 @@
 extends Control
 
+var game_paused : bool = false:
+	get:
+		return game_paused
+	set(value):
+		game_paused = value
+		get_tree().paused = game_paused
+		_update_visibility()
```

Comment:
To improve readability, what you think to do like this?:


```suggestion
var game_paused : bool = false : get = _get_game_paused, set = _set_game_paused

func _get_game_paused():
	return game_paused

func _set_game_paused(new_state: bool):
	game_paused = new_state
	get_tree().paused = game_paused
	_update_visibility()
```

What you think?

Reply:
I prefer the way I did it but if the team agrees I can change to what you recommended

Reply:
specially since that will contain this  `show() if game_paused else hide()` instead of the function

Reply:
Hmm, ok, I understand, but for consistency, I think it's better to do it this way, because imagine that eventually we have a set/get with 10 lines (or more), we don't want it to be next to the variable

Made sense? 😅

Reply:
Not 100% sure, but I think you don't need the `get` method :)

Reply:
Yap, it's not needed :)